package/kernel/mt76/Makefile

```diff
@@ -360,6 +360,8 @@ ifdef CONFIG_PACKAGE_kmod-mt7663u
 endif
 ifdef CONFIG_PACKAGE_kmod-mt7915e
   PKG_MAKE_FLAGS += CONFIG_MT7915E=m
+  PKG_MAKE_FLAGS += CONFIG_MT76_LEDS=y
+  NOSTDINC_FLAGS += -DCONFIG_MT76_LEDS
```

Comment:
For mt7615 this is activated in the Makefile:
https://github.com/openwrt/mt76/blob/master/mt7615/Makefile#L3

@nbd168  should we activate this also for mt7915 in the mt7915 Makefile?

Reply:
Should I make a request in mt76 and revert this?

Reply:
Done, submitted a PR in mt76, removed the changes from here.

Reply:
Thanks, I hope @nbd168  will notice it there.